package.json

```diff
@@ -9,16 +9,25 @@
   },
   "scripts": {
     "start": "./entry.sh",
-    "build": "coffee -c src",
+    "build": "coffee -c src && webpack --optimize-minimize --define process.env.NODE_ENV=\"production\"",
```

Comment:
Can't these options be put in `webpack.config.js` so we have all the webpack config in one place?